build/proj/GenerateDependentAssemblyVersionFile.proj

```diff
@@ -1,46 +0,0 @@
-<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
```

Comment:
Replaced this with CreateDependentAssemblyVersionsCsv in setup/Directory.Build.targets